tests/testthat/test-pstr_aggregate_scores.R

```diff
@@ -7,3 +7,16 @@ test_that("FIXME? Do we intend to return a grouped data frame?", {
 
   expect_true(dplyr::is_grouped_df(out))
 })
+
+test_that("outputs a result for companies without any products", {
+  companies <- pstr_toy_companies('products' = NA)
+  ep_weo <- pstr_toy_ep_weo()
+  weo <- pstr_toy_weo_2022()
+
+  out <- companies |>
+    pstr_add_reductions(ep_weo, weo) |>
+    pstr_add_transition_risk() |>
+    pstr_aggregate_scores(companies)
+
+  expect_equal(FALSE, is.null(out$score_aggregated))
+})
```

Comment:
Thanks @lindadelacombaz 

The test looks great but the expectation seems too capture a too-narrow set of unexpected results. What if `score_aggregated` is `NA` or `999999`?

Maybe instead focus on what's expected? Is that `100`? 

So how about this?

```r
  expect_equal(out$score_aggregated, 100)
```

Reply:
oh yes. my mistake - I just changed it :)